test/resources/scan_form.js

```diff
@@ -0,0 +1,45 @@
+import apiStub from '../helpers/apiStub';
+import NotImplementedError from '../../src/errors/notImplemented';
+
+import scanForm from '../../src/resources/scan_form';
+
+describe('ScanForm Resource', () => {
+  it('exists', () => {
+    expect(scanForm).to.not.be.undefined;
+    expect(scanForm).to.be.a('function');
+  });
+
+  it('unwraps array responses', () => {
+    const ScanForm = scanForm(apiStub());
+    const data = [new ScanForm()];
+    expect(ScanForm.unwrapAll({ scan_forms: data })).to.deep.equal(data);
+  });
+
+  it('throws on delete', () => {
+    const ScanForm = scanForm(apiStub());
+    ScanForm.delete('id').then(() => {}, (err) => {
+      expect(err).to.be.an.instanceOf(NotImplementedError);
+    });
+  });
+
+  describe('toJSON', () => {
+    let ScanForm;
+    let stub;
+
+    beforeEach(() => {
+      stub = apiStub();
+      ScanForm = scanForm(stub);
+    });
+
+    it('returns the shipments if available', () => {
+      const shipments = { shipments: [{ id: 'shp_123' }] };
+      const sf = new ScanForm(shipments);
+      expect(sf.toJSON()).to.deep.equal(shipments);
+    });
+
+    it('calls the super if there\'s no shipments', () => {
+      const sf = new ScanForm();
+      expect(sf.toJSON()).to.deep.equal({});
```

Comment:
@ajacksified while this passes, it's not quite correct. it doesn't actually test that the super was called. Don't see any examples of that in the current tests.